test

```diff
@@ -169,9 +162,17 @@ function dep_tests {
 }
 
 # fail fast on static tests
+toggle_failpoints disable
```

Comment:
these tests don't pass with failpoints enabled since the code doesn't pass gofmt, so it forces it off

the CI isn't doing anything with failpoints yet